pyproject.toml

```diff
@@ -79,7 +79,7 @@ distance-dirty = "{base_version}+{distance}.{vcs}{rev}.dirty"
 method = "git"  # <- The method name
 # Parameters to pass to the method:
 match = ["*"]
-default-tag = "1.0.0"
+default-tag = "0.1.0"
```

Comment:
I'm actually not sure what this does? For `alchemiscale` we have it set to "0.0.0"...